routeros/resource_wifi_provisioning.go

```diff
@@ -54,20 +54,20 @@ func ResourceWifiProvisioning() *schema.Resource {
 			Description: "Regular expression to match radios by router identity.",
 		},
 		"master_configuration": {
-			Type:        schema.TypeString,
-			Optional:    true,
+			Type:     schema.TypeString,
+			Optional: true,
 			Description: "If action specifies to create interfaces, then a new master interface with its configuration " +
 				"set to this configuration profile will be created.",
 		},
 		"name_format": {
-			Type:         schema.TypeString,
-			Optional:     true,
-			Description:  "Specify the format of the CAP interface name creation.",
+			Type:        schema.TypeString,
+			Optional:    true,
+			Description: "Specify the format of the CAP interface name creation.",
 		},
 		"radio_mac": {
 			Type:         schema.TypeString,
 			Optional:     true,
-			Default:      "00:00:00:00:00:00",
+			Default:      "",
 			Description:  "MAC address of radio to be matched, empty MAC (00:00:00:00:00:00) means match all MAC addresses.",
 			ValidateFunc: ValidationMacAddress,
 		},
```

Comment:
I think the correct approach would be to ignore the value unless provided so that we can make it backward-compatible. No `Default` property in this case.

```suggestion
			Description:  "MAC address of radio to be matched, empty MAC (00:00:00:00:00:00) means match all MAC addresses.",
			ValidateFunc: ValidationMacAddress,
			DiffSuppressFunc: AlwaysPresentNotUserProvided,
		},
```

Reply:
I agree. I'll test it more today or tomorrow, I've discovered I had some issues when running provider after committing - especially when dealing with entries created with upstream. 

One thing I also noticed was the fact provider seemed to partially behave like WebFig: it didn't treat unsetting `radio-mac` as configuration drift if it was `00:00:00:00:00:00` before. I'll confirm it.

Reply:
@danielskowronski 
Daniel, can I make the changes that Michael suggested?